### ISSUE TITLE
VMagents doesn't scrap metrics after rolling update

### ISSUE THREAD

**Issue by alexandrst88** [VictoriaMetrics/operator#407]
Hi! we have current setups of vmagents

```yaml
apiVersion: operator.victoriametrics.com/v1beta1
kind: VMAgent
metadata:
  name: vmagent-ha
spec:
  image:
    tag: v1.66.0
  vmAgentExternalLabelName: vmagent_ha_prod
  additionalScrapeConfigs:
    name: additional-scrape-config
    key: additional-config.yaml
  replicaCount: 2
  shardCount: 3
  extraArgs:
    promscrape.maxScrapeSize: 48MB
  podMetadata:
    labels:
      env: prod
  resources:
    requests:
      cpu: 4
      memory: 8Gi
    limits:
      cpu: 4
      memory: 8Gi
  readinessProbe:
    httpGet:
      path: /health
      port: 8429
    successThreshold: 1
    failureThreshold: 10
    periodSeconds: 5
    timeoutSeconds: 5
    initialDelaySeconds: 180
```

But after rolling update(for example bumping vmagent version) we have 30-60s gaps on the graphs.
![Screenshot 2022-01-12 at 16 42 23](https://user-images.githubusercontent.com/8613199/149161980-2570a764-d79a-4d9b-a95a-83ecc0e4fd40.png)

**Comment by alexandrst88** [comment]
Does VM HA work at all? I tried to have 3 replicas without shards, killed one pod and scrapes line dropped off?

**Comment by valyala** [comment]
The `Scrape rate` graph at the [official dashboard for vmagent](https://docs.victoriametrics.com/vmagent.html#monitoring) shows the summary scrape rate across all the vmagent instances by default (if a particular vmagent instance isn't selected at the top of dashboard). So, if a single vmagent is restarted, then it is expected that the summary scrape rate drops until the vmagent is returned back online.

**Comment by alexandrst88** [comment]
@valyala it's related https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2013

**Comment by f41gh7** [comment]
Must be solved with v1.72.0 vmagent release.
